src/applications/discharge-wizard/components/v2/resultsComponents/OptionalStep.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  SHORT_NAME_MAP,
+  RESPONSES,
+} from '../../../constants/question-data-map';
+
+const OptionalStep = ({ formResponses }) => {
+  const validReason = [RESPONSES.REASON_1, RESPONSES.REASON_2].includes(
+    formResponses[SHORT_NAME_MAP.REASON],
+  );
+  const dischargeYear = parseInt(
+    formResponses[SHORT_NAME_MAP.DISCHARGE_YEAR],
+    10,
+  );
+  const validYear = dischargeYear >= 2001 && dischargeYear <= 2009;
+
+  if (validReason && validYear) {
+    return (
+      <section className="feature">
+        <h4>
+          (Optional): Apply to the Physical Disability Board of Review (PDBR)
+        </h4>
+        <p>
+          If you believe your disability rating for TBI, PTSD, or mental health
+          conditions is too low, consider applying to the Physical Disability
+          Board of Review (PDBR). The DoD created the PDBR specifically to
+          review appeals about low disability ratings for Veterans discharged
+          between 2001 and 2009. Some Veterans discharged during this period of
+          time received lower disability ratings than they deserved, especially
+          if they suffered from TBI, PTSD, or other mental health conditions. If
+          you were discharged during this period of time and previously received
+          a disability rating of 20% or lower, you’re eligible to apply to the
+          PDBR for review. The PDBR does not issue discharge upgrades and cannot
```

Comment:
Should the word "not" be bolded here?

<img width="882" alt="Screenshot 2024-08-06 at 2 16 45 PM" src="https://github.com/user-attachments/assets/860b1ded-df9d-4cc1-bd21-25755a3ba385">

Reply:
@thejordanwood One more question for you on this PR. This is not bold in prod but it was in the mural. Was this a change we wanted to implement?

Reply:
@chriskim2311 Let's stick with how it is on prod (not bold). I thought I copied the text styles directly from prod, but that could've been an error on my part or maybe it's been updated since I documented this last year.